Corona-Warn-App/schemas/de.rki.coronawarnapp.eventregistration.storage.EventRegistrationDatabase/1.json

```diff
@@ -0,0 +1,70 @@
+{
+  "formatVersion": 1,
+  "database": {
+    "version": 1,
+    "identityHash": "6570bad9213df22a0fe02a3f2185da8e",
+    "entities": [
+      {
+        "tableName": "hostedEvents",
+        "createSql": "CREATE TABLE IF NOT EXISTS `${TABLE_NAME}` (`guid` TEXT NOT NULL, `description` TEXT NOT NULL, `location` TEXT NOT NULL, `startTime` TEXT, `endTime` TEXT, `defaultCheckInLength` INTEGER, `signature` TEXT NOT NULL, PRIMARY KEY(`guid`))",
```

Comment:
what is `location` ?

Reply:
regarding the start and end time, do we have it as Text in other places or Long?

Reply:
`location` is now `address` (changed in the new tech spec)